spec/features/user_contributes_new_councillor_spec.rb

```diff
@@ -0,0 +1,128 @@
+require "spec_helper"
+
+feature "Contributing new councillors for an authority" do
+  let(:authority) { create(:authority, full_name: "Casey City Council") }
+
+  context "when the feature flag is off" do
+    it "isn't available" do
+      visit new_authority_councillor_contribution_path(authority.short_name_encoded)
+
+      expect(page.status_code).to eq 404
+    end
+  end
+
+  context "when the feature flag is on" do
+    around do |test|
+      with_modified_env CONTRIBUTE_COUNCILLORS_ENABLED: "true" do
+        test.run
+      end
+    end
+
+    scenario "on the contribution page" do
+      visit new_authority_councillor_contribution_path(authority.short_name_encoded)
+
+      expect(page).to have_content("Casey City Council")
+    end
+
+    it "works successfully when the contributor provides their information" do
+      visit new_authority_councillor_contribution_path(authority.short_name_encoded)
+
+      within ".councillor-contribution-councillors fieldset" do
+        fill_in "Full name", with: "Mila Gilic"
+        fill_in "Email", with: "mgilic@casey.vic.gov.au"
+      end
+
+      click_button "Submit"
+
+      within_fieldset "Please tell us about yourself, so we can send you a little note of appreciation and updates about your contribution when it goes live." do
+        fill_in "Name", with: "Jane Contributes"
+        fill_in "Email", with: "jane@contributor.com"
+      end
+
+      click_button "Submit"
+
+      expect(page).to have_content "Thank you"
+    end
+
+    it "works successfully without contributor information" do
```

Comment:
To be consistent with the test above, I recon the description here should be something like "works successfully when the contributor does not provide their information"